Sources/CombineFeedback/System.swift

```diff
@@ -19,6 +19,7 @@ extension Publishers {
             return Publishers.MergeMany(events)
                 .receive(on: scheduler)
                 .scan(initial, reduce)
+                .prepend(initial)
```

Comment:
I removed it because it did not work nice with SwiftUI, where it would not render  initial value. Anyway if someone wants to do that they can always do it manually 🤷‍♂️

Reply:
Ok, so if we remove this, then we have to adjust the tests to take this into account.

Reply:
> it did not work nice with SwiftUI, where it would not render initial value

Although in the Examples it seems to work fine now...

Reply:
It's because of [this](https://github.com/sergdort/CombineFeedback/blob/master/Sources/CombineFeedbackUI/Widget.swift#L13) `SwiftUI` ignores the first value anyway if it's emitted synchronously.

Reply:
So do you want me to remove the `.prepend(initial)` ? It seems like it's more correct for System to emit it, regardless of how SwiftUI uses it, and it doesn't seem to cause any problems with `CombineFeedbackUI`.

If I remove it, then all the tests need to be fixed to handle the fact that the initial value is never emitted.

Reply:
Yeah, I guess we can keep it, if it will cause issues we can always remove it 👍